akka-actor-typed-tests/src/test/scala/akka/actor/typed/scaladsl/StashSpec.scala

```diff
@@ -613,21 +613,23 @@ class UnstashingSpec extends ScalaTestWithActorTestKit("""
       probe.expectMessage("handled 4")
     }
 
+    /* FIXME, do inside an actor
```

Comment:
?

Reply:
I moved it to StashBufferSpec rather than to an actor so it had a context